### PR TITLE
Fix raw apple crash reports with threads = null

### PR DESCRIPTION
**Pull request by mitsuhiko**
In some cases it was possible that the raw crash report see a None
value for threads or exception in which case it was falling over
instead of silently failing.

Fixes SENTRY-2MZ

### CODE REVIEW

File: src/sentry/api/endpoints/event_apple_crash_report.py [hunk 1]
```diff
@@ -46,15 +46,8 @@ def get(self, request, event_id):
...
-        threads = event.data.get(
-            'sentry.interfaces.threads',
-            event.data.get('threads'
-        )).get('values')
-
-        exception = event.data.get(
-            'sentry.interfaces.Exception',
-            event.data.get('exception'
-        )).get('values')
+        threads = (event.data.get('threads') or {}).get('values')
```
Comment (mattrobenolt) [inline src/sentry/api/endpoints/event_apple_crash_report.py]: Why are these changing? Seems unrelated.

Comment (mitsuhiko) [inline src/sentry/api/endpoints/event_apple_crash_report.py]: this code can never actually encounter `sentry.interfaces.threads` which is wrong anyways (if anything it would be `sentry.interfaces.Threads`) or `exception` (which always is `sentry.interfaces.Exception`). So since I already touched that code i wanted to fix that.

**Review by mattrobenolt — APPROVED** [review]
